### PR TITLE
Update README.md

### LINKED ISSUE #2156: We should add a slack badge to our README
We have a slack in CNCF but link isn’t obvious.

### PR DESCRIPTION
**Pull request by octonawish-akcodes**
#### What type of PR is this?
/doc
#### What this PR does / why we need it:
This PR will add slack link of armada.
#### Which issue(s) this PR fixes:

Fixes #2156

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,3 +49,7 @@ We expect readers of the documentation to have a basic understanding of Docker a
 
 - [Docker overiew](https://docs.docker.com/get-started/overview/)
 - [Kubernetes overview](https://kubernetes.io/docs/concepts/overview/)
+
+## Discussion
+
+Get engage with us on [Armada slack](https://cloud-native.slack.com/?redir=%2Farchives%2FC03T9CBCEMC)
```
Comment (kannon92) [inline README.md:55]:
```suggestion
If you are interested in discussing Armada you can find us on [Armada slack](https://cloud-native.slack.com/?redir=%2Farchives%2FC03T9CBCEMC).
```

Comment (kannon92) [reply README.md:55]: Could you also look into how to add github badges for slack?

**Review by kannon92 — COMMENTED** [review]
Thank you for your contribution! I have a few small comments.

File: README.md (hunk 1 above)
Comment (octonawish-akcodes) [reply README.md]: @kannon92 Done and I added the GitHub badge that is redirecting to cncf armada slack discussion's channel as suggested by you.

**Review by severinson — APPROVED** [review]

**Pull request merged**